src/pages/workspace/tags/TagSettingsPage.tsx

```diff
@@ -71,6 +75,10 @@ function TagSettingsPage({route, navigation}: TagSettingsPageProps) {
     };
 
     const updateWorkspaceTagEnabled = (value: boolean) => {
+        if (policy?.requiresTag && getCountOfEnabledTagsOfList(policyTagLists.at(0)?.tags) === 1 && !value) {
+            setShowCannotDisableLastTagModal(true);
+            return;
+        }
```

Comment:
Why are we checking the first tag list? Also how does this behave on multi-level tags?